docs/bundles/plain.md

```diff
@@ -2,21 +2,52 @@
 
 ## Overview
 
-This document is meant to define the `plain` bundle format as a reference for those publishing `plain` bundles
-for use with RukPak. For more information on the concept of a bundle, click [here](https://github.com/operator-framework/rukpak#bundle).
+This document is meant to define the plain bundle format as a reference for those publishing plain bundles for use with
+RukPak. A bundle is a collection of Kubernetes resources that are packaged together for the purposes of installing onto
+a Kubernetes cluster. A bundle can be unpacked onto a running cluster, where controllers can then create the underlying
+content embedded in the bundle. The bundle can be used as the underlying `spec.source` for
+a [Bundle](https://github.com/operator-framework/rukpak#bundle) resource.
 
-A plain bundle is simply a collection of static, arbitrary, Kubernetes YAML manifests in a given directory.
+A plain bundle is simply a collection of static, arbitrary, Kubernetes YAML manifests in a given directory. A plain
+bundle can consist of a container image, a directory in a git repository, or any other content source that
+the [plain bundle provisioner](../provisioners/plain.md)
+supports.
 
 The currently implemented plain bundle format is the `plain+v0` format. The name of the bundle format, `plain+v0`
 combines the type of bundle (plain) with the current schema version (v0).
+The [plain bundle provisioner](../provisioners/plain.md)
+is able to source `plain+v0` bundles and install them onto a Kubernetes cluster.
 
 > Note: the plain+v0 bundle format is at schema version v0, which means it's an experimental format that is subject
 > to change.
 
+Supported source types for a plain bundle currently include the following:
+
+* A [container image](../sources/image.md)
+* A [directory in a git repository](../sources/git.md)
+* A [configmap](../sources/local.md)
+* An [upload](../sources/upload.md)
+
+Additional source types (such as a local volume or a generic URI-based resource) are on the roadmap. These source types
+all present the same content (a directory containing static Kubernetes YAML manifests) in a different way.
+
+## Common Terminology
+
+* `bundle` is a collection of files that define content to be deployed to a cluster
+* `bundle image` is a container image that contains a bundle within its filesystem
+* `bundle git repo` is a git repository that contains a bundle within a directory
+
 ## Example
 
-For example, below is an example of a file tree in a `plain+v0` bundle. It must have a `manifests` directory contains
-the Kubernetes resources required to deploy an application.
+For example, below is a minimal example of a Dockerfile that builds a `plain+v0` bundle image from a directory
+containing static Kubernetes manifests.
+
+```dockerfile
+FROM scratch
+COPY /manifests /manifests
+```
+
+where the given `manifests` directory contains the Kubernetes resources required to deploy an application, for example:
 
 ```tree
```

Comment:
does `tree` do anything ?

Reply:
I don't think so. addressed in https://github.com/operator-framework/rukpak/pull/642/commits/486faf8915e2f3f891ee769b99651e3039d052b1